boards/frdm-kw41z/board.c

```diff
@@ -79,8 +37,11 @@ void board_init(void)
     /* Start the RTT, used as time base for xtimer */
     rtt_init();
 
-    /* initialize and turn off LED3 (red onboard LED) */
+    /* initialize and turn off LEDs */
     gpio_init(LED0_PIN, GPIO_OUT);
     gpio_set(LED0_PIN);
-    /* frdm_nor_init(); */
+    gpio_init(LED1_PIN, GPIO_OUT);
+    gpio_set(LED1_PIN);
+    gpio_init(LED2_PIN, GPIO_OUT);
+    gpio_set(LED2_PIN);
```

Comment:
SAUL turns these on again when saul is initialized, this is for applications without SAUL